scripts/test_downgrade_from_tag.sh

```diff
@@ -158,7 +158,8 @@ wait_for_pg() {
     exit 1
 }
 
-VERSION=`echo ${UPDATE_FROM_TAG} | sed 's/\([0-9]\{0,\}\.[0-9]\{0,\}\.[0-9]\{0,\}\).*/\1/g'`
+# shellcheck disable=SC2001
```

Comment:
```suggestion
# shellcheck disable=SC2001 -- See if you can use ${variable//search/replace} instead.
```

I find it convenient to add the description of what is disabled. Not sure it works in this syntax though, maybe a new line is required.

Reply:
Judging from their docs, should work with a second # instead of dashes
```suggestion
# shellcheck disable=SC2001  # See if you can use ${variable//search/replace} instead.
```